### PR TITLE
Feature movie random page

### PR DESCRIPTION
**Pull request by Pyramate**
- added the route and popup to header to generate random movie
- added the filter section in all movies page

### CODE REVIEW

**Comment by lchastanet** [comment]
PR in main...

File: frontend/src/components/GenreFilter.jsx [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        color="#15141f"
+        textAlign="left"
+        pb={5}
+        borderBottom="1px solid #eee"
+        width="100%"
+      >
+        Rating
+      </Heading>
+      <RangeSlider
+        // eslint-disable-next-line jsx-a11y/aria-proptypes
```
Comment (lchastanet) [inline frontend/src/components/GenreFilter.jsx:88]: Not possible there

File: frontend/src/components/GenreFilter.jsx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+          <Box color="teal.400" />
+        </RangeSliderThumb>
+        <RangeSliderThumb boxSize={6} index={1}>
+          <Box color="teal.400" />
+        </RangeSliderThumb>
+      </RangeSlider>
+
+      {/* <Button colorScheme="facebook" onClick={resetFilter}>
+        Reset Filters
+      </Button> */}
```
Comment (lchastanet) [inline frontend/src/components/GenreFilter.jsx:156]: Commented code here

File: frontend/src/components/Navbar.jsx [hunk 3]
```diff
@@ -11,9 +12,10 @@ export default function Navbar() {
         <Button variant="ghost" _hover={{ bgColor: "#2B3543" }}>
           <Link to="/popular">Popular</Link>
         </Button>
-        <Button variant="ghost" _hover={{ bgColor: "#2B3543" }}>
-          <Link to="/movies">What should I Watch?</Link>
-        </Button>
+        {/* <Button variant="ghost" _hover={{ bgColor: "#2B3543" }}>
+          <Link to="/get-inspired">What should I Watch?</Link>
+        </Button> */}
```
Comment (lchastanet) [inline frontend/src/components/Navbar.jsx:17]: Commented code here

**Review by lchastanet — CHANGES_REQUESTED** [review]

File: frontend/src/components/GenreFilter.jsx [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+      setGenreFilter("");
+    }
+  };
+  const filterByRating = () => {
+    setRating(sliderValue);
+  };
+  //   const resetFilter = () => {
+  //     filterByRating(sliderValue([0, 10]));
+  //     filterByGenre();
+  //   };
```
[the remark this answers is not in the document]
Comment (Pyramate) [reply frontend/src/components/GenreFilter.jsx]: fixed

File: frontend/src/components/GenreFilter.jsx (hunk 1 above)
Comment (Pyramate) [reply frontend/src/components/GenreFilter.jsx]: fixed

File: frontend/src/components/GenreFilter.jsx (hunk 2 above)
Comment (Pyramate) [reply frontend/src/components/GenreFilter.jsx]: fixed

File: frontend/src/components/Navbar.jsx (hunk 3 above)
Comment (Pyramate) [reply frontend/src/components/Navbar.jsx]: fixed

**Review by lchastanet — APPROVED** [review]
Nickel 👌🏻

**Pull request merged**